src/coreclr/src/zap/zapinfo.cpp

```diff
@@ -502,50 +502,61 @@ void ZapInfo::CompileMethod()
     ULONG cCode;
 
 #ifdef ALLOW_SXS_JIT_NGEN
+    bool expectedAltJitFailure = false;
```

Comment:
This is a bit confusing, but as I understand it, `compileMethod` always returns `CORJIT_SKIPPED` for an altjit compile.

If that's the issue, it might be clearer to call this `skippedAltJitCompile` as it's not *really* a failure. Also, a comment explaining this situation would be really useful.